### ISSUE TITLE
Re. stackup bundler

### ISSUE THREAD

**Issue by salzbrenner** [wevm/prool#17]
I came upon this repo after searching for success stories w/ anvil and stackup bundler. I've tried to get stackup's bundler working with anvil in the past, but ran into issues sending user operations w/ anvil's debug_traceCall support.

From anvil [docs](https://book.getfoundry.sh/reference/anvil/):
```
debug_traceCall Note that non-standard traces are not yet supported. This means you can’t pass any arguments to the trace parameter.
```

I'm curious if stackup's bundler actually works against anvil for user operations?

**Comment by moldy530** [comment]
`debug_traceCall` isn't handled well within anvil and all of the bundlers here use it for simulation. The alchemy bundler (rundler) which is supported here as well runs in `unsafe` mode when used with prool, which ultimately skips simulation. That does mean that if your account violates any of the rules being checked during validation, that won't be caught in your tests. That's an ok tradeoff to make for most of the cases where you're developing against anvil + a local bundler because that logic is more core the bundlers themselves and most likely not related to your local development.

**Comment by salzbrenner** [comment]
yeah, i've seen various issues around `debug_traceCall` in anvil's repo, just wanted to make sure I wasn't missing something and that it actually was supported.

Unless stackup's bundler has been updated, no they don't support `unsafe` mode, I've asked the stackup team about this before.

Our solution was to swap out anvil with a local geth node to get it working w/ stackup's bundler.
